apps/admin-gui-e2e/src/e2e/admin/perun-blocked-logins.cy.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+describe("Perun admin blocked logins management with role Perun admin", () => {
+  const testLogin = "test_blocking_login"
+
+
+  before(() => {
+    cy.login('', 'perun');
+  });
+
+  beforeEach(() => {
+    cy.visit('home')
+      .get('[data-cy=admin-button]')
+      .click();
+  });
+
+  it("test display blocked logins page", () => {
+    cy.get('[data-cy=blocked-logins]')
+      .click({force: true})
+      .url()
+      .should('include', '/blocked_logins');
+  })
+
+  it('test block login', () => {
+    cy.get('[data-cy=blocked-logins]')
+      .click({force: true})
+      .intercept('**/usersManager/blockLogins**')
+      .as('blockLogin')
+      .intercept('**/usersManager/getBlockedLoginsPage**')
+      .as('getBlockedLogins')
+      .get('[data-cy=block-logins-button]')
+      .click({force: true})
+      .get('[data-cy=logins-input]')
+      .type(testLogin, {force: true})
+      .get('[data-cy=submit-blocked-logins-button]')
+      .click()
+      .wait('@blockLogin')
+      .wait('@getBlockedLogins')
+      // assert that the login is listed as blocked
+      .get('[data-cy=filter-input]')
+      .type(testLogin, {force: true})
+      .get(`[data-cy=${testLogin}-checkbox]`)
+      .should('exist');
+  });
+
+  it('test unblock login', () => {
+    cy.get('[data-cy=blocked-logins]')
+      .click({force: true})
+      .intercept('**/usersManager/unblockLoginsById**')
+      .as('unblockLogins')
+      .intercept('**/usersManager/getBlockedLoginsPage**')
+      .as('getBlockedLogins')
+      .get('[data-cy=filter-input]')
+      .type(testLogin, {force: true})
```

Comment:
Each test should be independent. Instead of blocking login in the first test and unblocking it in the second, you should insert a login for the unblocking test directly into the DB (GitLab repository).